### PR TITLE
IT-4714 | ToggleButton

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: lib/js/components/Buttons/ToggleButton/ToggleButton.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+
+import ToggleButton from 'design-system/lib/js/components/Buttons/ToggleButton';
```
Comment (rogatty) [inline lib/js/components/Buttons/ToggleButton/ToggleButton.stories.mdx:3]: Shouldn't it be `./...` as in other stories?

File: lib/js/components/Buttons/ToggleButton/ToggleButton.vue [hunk 2]
```diff
@@ -0,0 +1,322 @@
+<template>
+	<ds-ripple
+		class="toggleButtonWrapper"
+		:disable="!isInteractive"
+		:color="rippleColor"
+		@click="$emit('click')"
```
Comment (rogatty) [inline lib/js/components/Buttons/ToggleButton/ToggleButton.vue:6]: I believe it doesn't do anything.
```suggestion
```

**Comment by rogatty** [comment]
Ripple doesn't work properly (https://bethinkteam.slack.com/archives/CFBLF5RGA/p1660207832808779?thread_ts=1659692857.447919&cid=CFBLF5RGA) but apart from that and some minor cleanup, it LGTM.

**Review by rogatty — APPROVED** [review]

File: lib/js/components/Buttons/ToggleButton/ToggleButton.vue (hunk 2 above)
Comment (kvas-damian) [reply lib/js/components/Buttons/ToggleButton/ToggleButton.vue:9]: with this line you don't have to attach event handlers as `native` when using this component

File: lib/js/components/Buttons/ToggleButton/ToggleButton.stories.mdx (hunk 1 above)
Comment (kvas-damian) [reply lib/js/components/Buttons/ToggleButton/ToggleButton.stories.mdx]: [`6bd2ba0` (#176)](https://github.com/bethinkpl/design-system/pull/176/commits/6bd2ba06f9921c0da22b5a2cff4582994e786564)

**Comment by kvas-damian** [comment]
> Ripple doesn't work properly ([bethinkteam.slack.com/archives/CFBLF5RGA/p1660207832808779?thread_ts=1659692857.447919&cid=CFBLF5RGA](https://bethinkteam.slack.com/archives/CFBLF5RGA/p1660207832808779?thread_ts=1659692857.447919&cid=CFBLF5RGA)) but apart from that and some minor cleanup, it LGTM.

Fixed in [`58e5e28` (#176)](https://github.com/bethinkpl/design-system/pull/176/commits/58e5e282ab633be7a449b7db129f1796b3b7f4ea)

File: lib/js/components/Buttons/ToggleButton/ToggleButton.vue (hunk 2 above)
Comment (rogatty) [reply lib/js/components/Buttons/ToggleButton/ToggleButton.vue:9]: But you already have https://github.com/bethinkpl/design-system/blob/5350d819d6ea7b00a3591265f9daef9213de0e94/lib/js/components/Buttons/ToggleButton/ToggleButton.vue#L29

Comment (kvas-damian) [reply lib/js/components/Buttons/ToggleButton/ToggleButton.vue:9]: fotgot about it :facepalm:

**Pull request merged**